HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs

```diff
@@ -87,7 +88,12 @@ public List<SurveyResult> GetSurveyResultsAboutMedicalStaff()
 
         public List<SurveyAboutDoctor> GetSurveysByDoctor(string jmbg)
         {
-            return _context.SurveysAboutDoctor.Where(x => x.DoctorJmbg.Equals(jmbg)).ToList();
+            return _context.SurveysAboutDoctor.Where(x => GetExaminationsIdByDoctor(jmbg).Contains(x.ExaminationId)).ToList();
```

Comment:
Examination has the DoctorJmbg field, why do this instead of just using that field?

Reply:
Fixed in 7ae66b5349986db657ee712ed782bf8d87fc9033.